src/AutoRest.CSharp/Common/Input/InputTypes/InputType.cs

```diff
@@ -3,4 +3,4 @@
 
 namespace AutoRest.CSharp.Common.Input;
 
-internal abstract record InputType(string Name, bool IsNullable = false) { }
+internal abstract record InputType(string Name, bool IsLowConfident, bool IsNullable = false) { }
```

Comment:
Can `IsLowConfident` has a default value `false` so that by default we will not set it, only when it has `IsLowConfident` value in the json, we will set it?

Reply:
as I said in [this comment](https://github.com/Azure/autorest.csharp/pull/3354#discussion_r1270184726) below, using optional properties here is not quite a good idea, it could lead to hidden problems.
But I agree we could omit the values in the tspCodeModel file if it is using its default values. But I do not see quite much value for that